src/Sql/dbo/Tables/ProviderUser.sql

```diff
@@ -0,0 +1,14 @@
+﻿CREATE TABLE [dbo].[ProviderUser] (
+    [Id]           UNIQUEIDENTIFIER    NOT NULL,
+    [ProviderId]   UNIQUEIDENTIFIER    NOT NULL,
+    [UserId]       UNIQUEIDENTIFIER    NULL,
+    [Email]        NVARCHAR (256)      NULL,
+    [Key]          VARCHAR (MAX)       NULL,
+    [Status]       TINYINT             NOT NULL,
+    [Type]         TINYINT             NOT NULL,
+    [CreationDate] DATETIME2 (7)       NOT NULL,
```

Comment:
Will likely also need Permissions here for this relationship. Note we'll likely want to follow the custom permissions paradigm laid out for OrganizationUser vs. hard-coding inferred permissions to a given Type. Different providers will have different needs and ways they've decided to organize their users around managing orgs and services.

Reply:
Do we still want to keep `Type` here as an easy way to differentiate between Owners and Members, or just use Permissions?

Reply:
Yes, keep Type but add Permissions